cmd/kite/alerts/cmd.go

```diff
@@ -54,7 +54,7 @@ func init() {
 		&options.assignment,
 		"assigned-to",
 		"self",
-		"Filter alerts based on user or team",
+		"View alerts assigned to a specific team, your logged in PagerDuty user account (default), or the Silent Test user.",
```

Comment:
The phrase 'specific team' is misleading here, it is indicating that we can use a specific team name as a flag here even if it is not the one we have selected for our account. That's not applicable here. Please read the JIRA description properly and then modify this.